samples/bluetooth/broadcast_config_tool/src/main.c

```diff
@@ -235,6 +239,91 @@ static void subgroup_send(struct stream_index stream_idx)
 	prev_ret = ret;
 }
 
+static int dummy_data_config(const struct shell *shell, uint8_t big_index, uint8_t sub_index,
+			     uint8_t bis_index)
+{
+	int ret;
+
+	if (sd_card_not_present) {
+		LOG_DBG("No SD card, using dummy data");
+	} else {
+		shell_error(shell, "Failed as SD card present");
+		return -EINVAL;
+	}
+
+	if (broadcast_source_is_streaming(big_index)) {
+		shell_error(shell, "Cannot assign data while already streaming");
+		return -EFAULT;
+	}
+
+	if ((bis_index >= CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT) ||
+	    (bis_index >= broadcast_param[big_index].subgroups[sub_index].num_bises)) {
+		shell_error(shell, "BIS index %d is out of range (max %d)", bis_index,
+			    broadcast_param[big_index].subgroups[sub_index].num_bises - 1);
+		return -EINVAL;
+	}
+
+	LOG_DBG("Selecting dummy data for stream big: %d sub: %d bis: %d", big_index, sub_index,
+		bis_index);
+
+	struct bt_audio_codec_cfg *codec_cfg =
+		&broadcast_param[big_index].subgroups[sub_index].group_lc3_preset.codec_cfg;
+
+	struct lc3_stream_cfg cfg;
+
+	ret = le_audio_freq_hz_get(codec_cfg, &cfg.sample_rate_hz);
+	if (ret) {
+		shell_error(shell, "Failed to get frequency: %d", ret);
+		return ret;
+	}
+
+	ret = le_audio_duration_us_get(codec_cfg, &cfg.frame_duration_us);
+	if (ret) {
+		shell_error(shell, "Failed to get frame duration: %d", ret);
+	}
+
+	ret = le_audio_bitrate_get(codec_cfg, &cfg.bit_rate_bps);
+	if (ret) {
+		shell_error(shell, "Failed to get bitrate: %d", ret);
+	}
```

Comment:
What are these used for?

Reply:
I was having these if we were going to use Simple's coded data, but not needed here for the dummy data. Will remove as below.

Reply:
Removed